modules/docs/docgen/spec/docParser.spec.ts

```diff
@@ -635,6 +648,7 @@ describe('docParser', () => {
         export type Bar<T> = {value: T}
       `);
       const docs = parse(program, 'test.ts');
+
```

Comment:
these lines between test setup and assertions are always appreciated

Reply:
I'm not sure how they were removed. I made sure to add them